internal/app/server/open_saves.go

```diff
@@ -143,12 +144,11 @@ func (s *openSavesServer) DeleteRecord(ctx context.Context, req *pb.DeleteRecord
 }
 
 func (s *openSavesServer) GetStore(ctx context.Context, req *pb.GetStoreRequest) (*pb.Store, error) {
-	store, err := s.metaDB.GetStore(ctx, req.GetKey())
+	str, err := s.getStoreAndCache(ctx, req.GetKey())
 	if err != nil {
-		log.Warnf("GetStore failed for store (%s): %v", req.GetKey(), err)
```

Comment:
Can we leave some debug logs here (log.Debug)?

Reply:
we have debugs also on the `getStoreAndCache` we'll see it when the store cache is hit or missed.